### ISSUE TITLE
Can we get rid of it nagging for donation key when you run /co status?

### ISSUE THREAD

**Issue by a8underscore** [ParadauxIO/CoreProtect#1]
example:
![image](https://user-images.githubusercontent.com/60801647/108917025-f39b4f80-75e3-11eb-879f-a26128817846.png)

**Comment by ParadauxIO** [comment]
Yes, absolutely! Will work on it, or you can submit a PR whichever comes first
